usr/lib/tpm_stdll/tpm_specific.c

```diff
@@ -2619,6 +2668,53 @@ CK_RV token_specific_des_cbc(STDLL_TokData_t * tokdata,
         rc = CKR_OK;
     }
     return rc;
+#else
+    const EVP_CIPHER *cipher = EVP_des_cbc();
+    EVP_CIPHER_CTX *ctx = NULL;
+    CK_ATTRIBUTE *attr = NULL;
+    unsigned char dkey[8];
+    CK_ULONG rc;
+    int outlen;
+
+    UNUSED(tokdata);
+
+    // get the key value
+    if (template_attribute_find(key->template, CKA_VALUE, &attr) == FALSE) {
+        TRACE_ERROR("template_attribute_find(CKA_VALUE) failed.\n");
+        return CKR_FUNCTION_FAILED;
+    }
+
+    memcpy(dkey, attr->pValue, sizeof(dkey));
+
+    if (in_data_len % 8 || in_data_len > INT_MAX) {
```

Comment:
same here, use some define or for the 8 byte DES block size.

Reply:
done